### PR TITLE
Adding Platinum Pedigree to registry

### PR DESCRIPTION
**Pull request by zeeev**
Making the datasets public.

### CODE REVIEW

File: datasets/platinum-pedigree.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - pedigree
+  - sequencing data
+  - PacBio
+  - ONT
+  - Illumina
+  - Strand-seq
+License: 
+Resources:
+  - Description: https://github.com/Platinum-Pedigree-Consortium/Platinum-Pedigree-Datasets
+    ARN: arn:aws:s3:::platinum-pedigree-data
```
Comment (zeeev) [inline datasets/platinum-pedigree.yaml:21]: Hi @berylrab can you check our ARN?

**Comment by kc00k** [comment]
@zeeev the build is failing. Could you please take a look at this?

File: datasets/platinum-pedigree.yaml (hunk 1 above)
Comment (berylrab) [reply datasets/platinum-pedigree.yaml:21]: Hi @zeeev the ARN looks ok.

**Comment by berylrab** [comment]
Hey Zev, please use the tags that are already in the [tags.yaml](https://github.com/awslabs/open-data-registry/blob/main/tags.yaml). We typically do not add tags unless they are used by several datasets. Some of them need to match the ones we already have, it needs to be "genetic", "genomic", "life sciences", "whole genome sequencing". You do not need to add a tag for PacBio, this would already be covered under "Managed By".

**Review by berylrab — APPROVED** [review]

**Pull request merged**